railties/lib/rails/generators/rails/encrypted_secrets/encrypted_secrets_generator.rb

```diff
@@ -0,0 +1,66 @@
+require "rails/generators/base"
+require "rails/secrets"
+
+module Rails
+  module Generators
+    class EncryptedSecretsGenerator < Base
+      def add_secrets_key_file
+        unless File.exist?("config/secrets.yml.key") || File.exist?("config/secrets.yml.enc")
```

Comment:
@kaspth Currently, running `rails secrets:setup` after you have already run it once result in "nothing":

```sh
$ rails secrets:setup
$
```

How do you feel about raising an exception/warning if someone calls `rails secrets:setup` but the `secrets.yml.key` and/or `secrets.yml.enc` already exist?

The message could say something like "secrets already found, use `rails secrets:edit` to edit them.